### ISSUE TITLE
Consider adding react v18 to peerDependencies

### ISSUE THREAD

**Issue by scraggo** [4lejandrito/react-guitar#31]
react v18 isn't included in the list of `peerDependencies`.

Haven't fully tested, but it's working after using it in a newly installed `create-react-app` which uses react@18.2.0.

Not a blocking issue, but I need to use a workaround like `--legacy-peer-deps` in order to use the packages.

```sh
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: react-guitar@1.1.1
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react

...

npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8 || ^17.0" from react-guitar@1.1.1
npm ERR! node_modules/react-guitar
npm ERR!   react-guitar@"^1.1.1" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^16.8 || ^17.0" from react-guitar@1.1.1
npm ERR!   node_modules/react-guitar
npm ERR!     react-guitar@"^1.1.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

If I can help, let me know.

Thank you for considering this.

**Comment by 4lejandrito** [comment]
Thanks @scraggo! I'll do during the weekend.

**Comment by 4lejandrito** [comment]
Done! Published as `1.1.2`.

**Comment by scraggo** [comment]
Works great! Thanks for doing this so quickly 🥇
